web/forms.py

```
@@ -276,6 +277,51 @@
+            "image": TailwindFileInput(
+                attrs={
+                    "accept": "image/*",
+                }
+            ),
+        }
+
+    help_texts = {
+        "itemized_budget": "Enter a JSON object with budget items and costs",
+        "video_url": "Link to a video that explains your campaign",
+        "image": "Upload an image for your campaign (max size: 5MB)",
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)